### PR TITLE
release(Qcert) coq-qcert version 1.4.0.

### PR DESCRIPTION
**Pull request by jeromesimeon**
Signed-off-by: Jerome Simeon <jeromesimeon@me.com>

### CODE REVIEW

File: released/packages/coq-qcert/coq-qcert.1.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "coq-qcert"
+version: "1.4.0"
```
Comment (clarus) [inline released/packages/coq-qcert/coq-qcert.1.4.0/opam]: I think these fields could be removed as they are redundant with the package file name.

Comment (clarus) [reply released/packages/coq-qcert/coq-qcert.1.4.0/opam]: (I mean `name` and `version`)

Comment (jeromesimeon) [reply released/packages/coq-qcert/coq-qcert.1.4.0/opam]: I've removed `name` and `version` as suggested.

Comment (clarus) [reply released/packages/coq-qcert/coq-qcert.1.4.0/opam]: Thanks!

**Comment by jeromesimeon** [comment]
Thanks!

**Pull request merged**